Formula/elvish.rb

```diff
@@ -16,10 +16,12 @@ class Elvish < Formula
 
   def install
     ENV["GOPATH"] = buildpath
+    version = `git describe --tags --always`.strip
+    goroot = `go env GOROOT`.strip
```

Comment:
This needs some elaboration as well.

Reply:
Yeah this isn't worth setting unless the software is actually functionally deficient without it.

Reply:
I was just trying to match what Elvish's Makefile is doing for pre-built HEAD binaries, which will also match the next stable release.